weasyl/test/resetpassword/test_prepare.py

```diff
@@ -1,87 +1,69 @@
 from __future__ import absolute_import
 
+import hashlib
+
 import pytest
-import arrow
 
 from weasyl import resetpassword
 from weasyl import define as d
 from weasyl.test import db_utils
-from weasyl.test.utils import Bag
 
 
 @pytest.mark.usefixtures('db')
-def test_false_returned_if_token_does_not_exist():
+def test_none_returned_if_token_does_not_exist():
     token = "testtokentesttokentesttokentesttokentesttokentesttokentesttokentesttokentesttokentesttokentest000000"
-    assert not resetpassword.prepare(token)
+    assert resetpassword.prepare(token) is None
 
 
 @pytest.mark.usefixtures('db')
-def test_true_returned_if_token_exists():
+def test_user_info_returned_if_token_exists():
+    email_addr = "test@weasyl.com"
+    username = "checktoken0002"
+    user_id = db_utils.create_user(username=username, email_addr=email_addr)
+    token = "testtokentesttokentesttokentesttokentesttokentesttokentesttokentesttokentesttokentesttokentest000008"
```

Comment:
Technically I guess it doesn't matter since we're shoving everything into a SHA256 hash, and the controller/function don't care in the slightest of the token length, but ``request()`` would normally have the token be [25 characters long](https://github.com/Weasyl/weasyl/pull/787/files#diff-c674915a48e7f455260e394c4859d9acR23) via ``security.generate_key(25, ...)``. Just an observation.